src/components/componentHooks/useCurrentPriceSelector.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { quoteSelector } from "../../selectors/quoteSelector";
+import { createSelector } from "reselect";
+
+export const useCurrentPriceSelector = createSelector(
```

Comment:
This hook is already defined in quoteSelector.js, we can simply import it to this file and export the useSelector of it.

Reply:
You're right, that would be better.